src/sugar/autofill.ts

```diff
@@ -132,6 +131,19 @@ async function setNextValidSequenceNumber(
   tx.Sequence = data.result.account_data.Sequence
 }
 
+async function fetchAccountDeleteFee(client: Client): Promise<BigNumber> {
+  const response = await client.request({ command: 'server_info' })
+  console.log(response.result.info.validated_ledger)
+
+  const fee = response.result.info.validated_ledger?.reserve_inc_xrp
+
+  if (fee == null) {
+    return Promise.reject(new Error('Could not fetch Owner Reserve.'))
+  }
+
+  return new BigNumber(fee).multipliedBy(DROPS_PER_XRP)
```

Comment:
Why not just use `dropsToXrp` in `utils`?

Reply:
Better yet, use server_state instead to get the value in drops to begin with.